### ISSUE TITLE
Summarize [[this note]] not working

### ISSUE THREAD

**Issue by jack2game** [brianpetro/obsidian-smart-connections#656]
I've given the specific note in the conversation with Smart Chat using [[note]], but it still won't summarize it for me.

All notes have been embedded correctly.

![Untitled](https://github.com/brianpetro/obsidian-smart-connections/assets/16070158/e02ae023-b05f-4a47-80ab-c1d7140263e6)

**Comment by jack2game** [comment]
@christianngarcia thanks for looking into this.

I did a bit more research and it might be because the embeddings were not created correctly?

Here is a screenshot when I open the note that I want to summarize, and check the `Smart Connections` window, where it says `Block not found`.

However I did run the command `Refresh & Make Connections` multiple times, and even tried to refresh the entire library's embeddings, but still no luck.

If anything is worth mentioning, this note is not a short note - it is a transcription of a 2-hour meeting.

![Obsidian_1718265529_Anlm9eIR5G](https://github.com/brianpetro/obsidian-smart-connections/assets/16070158/bacb8147-c6c6-472a-b403-6d671aec01fe)
